website/docs/plugin/sdkv2/logging/http-transport.mdx

```diff
@@ -0,0 +1,211 @@
+|         `tf_http_req_body` | Request body                                                                                                                                                             |                                                                                                       |      Request       |
+|       `tf_http_req_method` | Request method                                                                                                                                                           | A canonical [HTTP methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Method)                  |      Request       |
+|          `tf_http_req_uri` | Request URI                                                                                                                                                              | Ex. `"/path"`                                                                                         |      Request       |
+|      `tf_http_req_version` | Request HTTP version                                                                                                                                                     | Ex. `"HTTP/1.1"`                                                                                      |      Request       |
+|         `tf_http_res_body` | Response body                                                                                                                                                            |                                                                                                       |      Response      |
+|  `tf_http_res_status_code` | Response status code                                                                                                                                                     | A canonical [HTTP status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)              |      Response      |
+| `tf_http_res_status_reason | Response status reason                                                                                                                                                   | Canonical textual description of the corresponding `tf_http_res_status_code`                          |      Response      |
+|      `tf_http_res_version` | Response HTTP version                                                                                                                                                    | Ex. `"HTTP/2.0"`                                                                                      |      Response      |
+|             (Other fields) | Request / Response headers. One field per header. If header contains a single value, field value will be set to that. Otherwise, field value will be a slice of strings. |                                                                                                       | Request / Response |
```

Comment:
```suggestion
|             (Other fields) | Request / Response headers. One field per header. If the header contains a single value, the log field value is set to that value. Otherwise, the field value is a slice of strings. |                                                                                                       | Request / Response |
```